src/components/Autocomplete/components/ComboBox/tests/ComboBox.test.tsx

```diff
@@ -458,6 +458,34 @@ describe('<ComboBox/>', () => {
       };
     });
 
+    it("will prevent default and select an item on the 'enter' keyup", () => {
```

Comment:
Instead of having the async arrow functions couldn't you just make this on `async` and just use `awaits` throughout?